.travis.yml

```diff
@@ -5,15 +5,24 @@ scala: 2.12.3
 
 jdk: oraclejdk8
 
-script: sbt ++$TRAVIS_SCALA_VERSION samples/compile test
+script:
+  - sbt ++$TRAVIS_SCALA_VERSION samples/compile test
+  - $(npm bin)/tsc samples/*.ts
```

Comment:
I think this should be `*.d.ts`.

Reply:
Fixed in https://github.com/sjrd/scala-js-ts-importer/pull/56/commits/b4dcd54d67a348a303379614e60640dec1e2794b